exercises/practice/hello-world/.meta/config.json

```
@@ -0,0 +1,12 @@
+{
+  "blurb": "The classical introductory exercise. Just say \"Hello, World!\"",
+  "authors": ["bushidocodes"],
+  "contributors": [],
+  "files": {
+    "solution": ["hello-world.wat"],
+    "test": ["hello-world.spec.js"],
+    "example": [".meta/proof.ci.js"]
```

```suggestion
    "example": [".meta/proof.ci.wat"]
```

See the below discussion.